custom/conf/app.example.ini

```diff
@@ -1262,6 +1262,12 @@ ROUTER = console
 ;; List of file extensions that should be rendered/edited as Markdown
 ;; Separate the extensions with a comma. To render files without any extension as markdown, just put a comma
 ;FILE_EXTENSIONS = .md,.markdown,.mdown,.mkd
+;;
+;; Enables math inline and block detection
+;ENABLE_MATH = true
+;;
+;; Enables in addition inline block detection using single dollars
+;ENABLE_INLINE_DOLLAR_MATH = false
```

Comment:
Maybe enable by default as well? Any downsides?

Reply:
There are lots of downsides for this one - KaTeX strongly recommends against this.

Reply:
Hmm okay, there's certainly risk for wrong interpretations.

Reply:
Do you have any links to KaTeX docs on this?

Reply:
Given Github have enabled this by default I think we have to do so too.